config/opal_config_pthreads.m4

```diff
@@ -251,9 +252,10 @@ AC_PROVIDE_IFELSE([AC_PROG_CC],
                   [OPAL_INTL_POSIX_THREADS_PLAIN_C],
                   [opal_pthread_c_success=1])
 
-AC_PROVIDE_IFELSE([AC_PROG_CXX],
-                  [OPAL_INTL_POSIX_THREADS_PLAIN_CXX],
-                  [opal_pthread_cxx_success=1])
+AS_IF([test "$WANT_MPI_CXX_SUPPORT" = 1],
```

Comment:
Should the test be
Is there a working c++ compiler ?
AND
Do we want to build c++ bindings ?

Which raises the question, how do we test if there is a working c++ compiler ?
```test -n $CXX``` ?
assuming AC_PROG_CXX was already invoked